translations/mod-fqm-manager/en.json

```diff
@@ -308,5 +308,20 @@
   "entityType.drv_holdings_record_details.holdings_temporary_location": "Holdings temporary location",
   "entityType.drv_holdings_record_details.holdings_temporary_location_id": "Holdings temporary location ID",
   "entityType.drv_holdings_record_details.holdings_statistical_code_ids": "Holdings statistical code ID list",
-  "entityType.drv_holdings_record_details.holdings_statistical_codes": "Holdings statistical code list"
+  "entityType.drv_holdings_record_details.holdings_statistical_codes": "Holdings statistical code list",
+
+  "entityType.src_organization_type": "Organization types",
+  "entityType.src_organization_type.id": "ID",
+  "entityType.src_organization_type.organization_types_name": "Type",
+  "entityType.drv_organization_details" : "Organizations - vendor info",
```

Comment:
Nit: it'd be nicer for this `-` to be replaced with an em-dash, it's longer for this kind of thing. You can copy it here: `—`

Reply:
You can see the difference when not monospace:

Organizations - vendor info
Organizations — vendor info

Reply:
you could also do a colon